application/controllers/admin/themes.php

```diff
@@ -425,6 +427,20 @@ public function uploadfile()
             $templatename           = returnGlobal('templatename');
             $oEditedTemplate        = Template::getInstance($templatename);
             $screenname             = returnGlobal('screenname');
+
+            // Check file size and redirect on error
+            $uploadHelper = new LSUploadHelper();
+            $uploadHelper->checkUploadedFileSizeAndRedirect(
+                'upload_file', 
+                array(
```

Comment:
For the future, feel free to use the `[]` shorthands instead of `array()` when creating arrays. Looks a bit more modern, I think. ^^